pkg/cryptoutils/privatekey.go

```diff
@@ -102,7 +102,7 @@ func UnmarshalPEMToPrivateKey(pemBytes []byte, pf PassFunc) (crypto.PrivateKey,
 	switch derBlock.Type {
 	case string(PrivateKeyPEMType):
 		return x509.ParsePKCS8PrivateKey(derBlock.Bytes)
-	case string(EncryptedCosignPrivateKeyPEMType):
+	case string(EncryptedSigstorePrivateKeyPEMType):
```

Comment:
Can we support both here? It's kinda ugly but it it decodes, it decodes.

Reply:
sgtm